tests/parm/input.mom6_ccpp.nml.IN

```diff
@@ -194,7 +194,7 @@
        tlives         = 1800
        nseed          = 1000000
        nfracseed      = 0.5
-       rcell          = 0.72
+       rcell          = 0.1
```

Comment:
In the new P7 tests, this is a parameter which matches the current P7 configuration, which is being run w/ 0.72. Is the value of rcell=0.1 something you'd like to change for the P8?

Reply:
@lisa-bengtsson can you comment on this?  All I know is that at 1-degree, which is the resolution that the RT, a rcell of 0.72 does not activate the ca_sgs scheme, so the RT doesn't actually properly exercise the ca_sgs code.

Reply:
Yes, sorry this got complicated. I'll try to explain; I did most of the testing at C768, then rcell=0.72 was found to be the optimum value. However, it turns out the impact doesn't scale well across resolutions even though I had tried to make it scale independent. So for the RT tests at C96 there could be cases where the CA never exceeded this threshold parameter and thus didn't have an impact. I am working on an update (I have created an issue for this), but in the meantime, for the RT tests at C96 I would propose to have rcell = 0.1. For the prototype P7 test I suggest to keep rcell=0.72 until I have committed my fix for this for P8.

Reply:
Alternatively keep 0.1 in all tests that just looks at the technical impact (reproducibility etc.) and keep 0.72 in the P7 production runs.

Reply:
Thanks Lisa. Maybe it makes sense to do what we do with the CDMBWD setting:

export CDMBWD_c96='0.14,1.8,1.0,1.0'
export CDMBWD_c192='0.23,1.5,1.0,1.0'

# set default
export CDMBWD=${CDMBWD_c96}

The higher resolution tests would use RCELL=RCELL_C384 =.72  but the low res tests we use just for regression testing would use RCELL=RCELL_C96=0.1 ?

Reply:
@lisa-bengtsson I'm going to put in the resolution dependence rcell values. Right now I have:

# CA tunable parameter
export RCELL_c46 =0.1
export RCELL_c96 =0.1
export RCELL_c192 =0.72
export RCELL_c384 =0.72
export RCELL_c768 =0.72

Do you have any suggestions for c192?

Reply:
Hi Phil and Denise, yes you can do this temporarily if you want, my PR will update this for P8. For c192 you can put 0.2, thanks.

Reply:
will do